node/src/command.rs

```diff
@@ -84,17 +91,98 @@ pub fn run() -> Result<()> {
 
                 runner.sync_run(|config| cmd.run::<Block, Executor>(config))
             } else {
-                println!(
-                    "Benchmarking wasn't enabled when building the node. \
+                Err("Benchmarking wasn't enabled when building the node. \
 				You can enable it with `--features runtime-benchmarks`."
-                );
-                Ok(())
+                    .into())
             }
         }
-        Some(Subcommand::Base(subcommand)) => {
-            let runner = cli.create_runner(subcommand)?;
-
-            runner.run_subcommand(subcommand, |config| Ok(new_full_start!(config).0))
+        Some(Subcommand::Key(cmd)) => cmd.run(),
```

Comment:
I tried to test the `key` command and `inspect` command. According to changes in https://github.com/paritytech/substrate/releases/tag/v2.0.0  the `inspect-key` subcommand was renamed to `inspect` but I still get the following message: 

```
error: The subcommand 'inspect' wasn't recognized
	Did you mean 'inspect-key'?

If you believe you received this message in error, try re-running with 'joystream-node key -- inspect'

USAGE:
    joystream-node key <SUBCOMMAND>
```

Reply:
I'm not sure of your initial command, but this commands work in my local enviroment:
```
./target/release/joystream-node inspect --dev block 1
./target/release/joystream-node key inspect-key
```

Reply:
Ah perhaps I mixed up the subkey command name with the subcommands for the node..

I tried:

```
./target/release/joystream-node key inspect
```

And as you pointed out the rename of `inspect-key` to `inspect` was for subkey utility not the node client :+1: